pyproject.toml

```diff
@@ -71,6 +71,7 @@ pysftp = "^0.2.9"
 dropbox = "^11.20.0"
 aiohttp-middlewares = "^2.0.0"
 opencolorio = "^2.2.0"
+ratelimiter = "^1.2.0.post0"
```

Comment:
What is this for?

Reply:
This is for the rate limiting. Clockify currently supports 10 requests per second. 
I guess this [function ](https://github.com/ynput/OpenPype/pull/4607/files#diff-77e6095f1cc4654b5d018ce050a6ccf789596314a1cf168835573a609477a2c2L15) was there for the same reason, although it is not clear how does it work (or whether it works at all) and also it lacks settings.

Reply:
> and also it lacks settings.

It still lacks them.

> although it is not clear how does it work

Well using RateLimiter did not help much to be honest.

Dependency which looks like is not maintained to add one functionality is complication. We also need to limit dependencies as much as possible as the clockify module will be soon separated and less dependencies means less issues on deployment.

Reply:
> It still lacks them.

By settings I meant some constants that would set the period and max retries. Sorry for not being clear. If these constants are set to some reasonable values, the result is definitely visible, especially for the loop that is checking for currently running timers. So do you suggest to remove it at all and hope this won't have any issues with the rate limiting, or rather incorporate it somewhere inside the module?